packages/ckeditor5-language/docs/features/language.md

```diff
@@ -24,6 +24,31 @@ There are other language-related CKEditor 5 features you may want to check:
 * {@link features/ui-language UI Language}  &ndash; Set the UI language.
 * {@link features/spelling-and-grammar-checking Spelling and grammar checking} &ndash; Employ multi-language spell check for flawless content.
 
+## Configuring available languages
+
+To modify the list of available languages displayed in the language dropdown, the {@link module:core/editor/editorconfig~LanguageConfig language config} needs to be updated.
```

Comment:
Wouldn't it be better to just say:

```
To modify the list of available languages displayed in the language dropdown use the @link module:core/editor/editorconfig~LanguageConfig#textPartLanguage `config.language.textPartLanguage`} configuration option.
```

This way we have one paragraph less and reader knows exactly what property should be used to customize the behavior?

cc **@godai78**

Reply:
Yes, this DOES sound better.